validator/src/main/java/io/avaje/validation/core/ResourceBundleManager.java

```diff
@@ -37,7 +40,13 @@ final class ResourceBundleManager {
   }
 
   private void addBundle(final String name, final Locale locale) {
-    map.computeIfAbsent(locale, key -> new ArrayList<>()).add(getBundle(name, locale));
+
+    try {
+      map.computeIfAbsent(locale, key -> new ArrayList<>()).add(getBundle(name, locale));
+
+    } catch (MissingResourceException e) {
+      logger.log(Level.ERROR, "failed to load " + name + "With locale " + locale);
```

Comment:
It looks like there needs to be a space character before `"With locale "` after `+ name +`